src/clr-angular/data/datagrid/datagrid-tabs.integration.spec.ts

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
This test belongs in `datagrid.spec.ts` that's where we do integration testing for datagrid where we need to do more than a pure unit test on a datagrid component. Also, I'm not a fan of adding the `*.integration.spec` filename to the project.

Reply:
Moved!